platform/platform-app/pom.xml

```diff
@@ -429,14 +430,14 @@
         <cxf.commons-lang.version>2.6</cxf.commons-lang.version>
         <cxf.commons-lang3.version>3.5</cxf.commons-lang3.version>
         <cxf.dropwizard.version>3.1.2</cxf.dropwizard.version>
-        <cxf.ehcache.version>2.10.3</cxf.ehcache.version>
+        <cxf.ehcache.version>2.10.4</cxf.ehcache.version>
         <cxf.fastinfoset.bundle.version>1.2.13_1</cxf.fastinfoset.bundle.version>
-        <cxf.guava.version>18.0</cxf.guava.version>
+        <cxf.guava.version>${guava.version}</cxf.guava.version>
```

Comment:
❓ Are these going to remain in lockstep going forward? Or might we or CXF want to upgrade guava causing an issue? And if that doesn't matter, should we get rid of this copy of the version property?

Reply:
Yes, because CXF is closely tied to DDF's security framework they should remain in lockstep. I think we'd want to keep the cxf version property as it makes doing upgrades easier because we can just copy and paste the CXF features straight from the CXF features file into DDF's feature file.

Reply:
Just to clarify, why does it matter that the cxf features use the same exact guava version as ddf? What are the consequences of the versions being different?

Reply:
There are security bundles in DDF that embed CXF classes, and at the same time those security bundles bring in the guava dependency. If the version of guava that those CXF classes bring into those bundles are different than the version of guava that is brought in by DDF, then a version conflicting error would. You can't have multiple versions of the same dependency brought into the same bundle because OSGI won't know how to wring things up correctly.

Reply:
Thanks for the explanation! Should <guava.version> also be added to the same section https://github.com/codice/ddf/pull/2739#discussion_r155321926 then?

Reply:
Yup, fixed.